miner/Cargo.toml

```diff
@@ -30,7 +29,7 @@ ckb-verification = { path = "../verification" }
 indicatif = "0.11"
 console = "0.7.5"
 ckb-dao = { path = "../util/dao" }
-ckb-hash = { path = "../util/hash"}
+eaglesong = { git = "https://github.com/rink1969/eaglesong", version = "0.1" }
```

Comment:
I asked @rink1969 if he could publish it, and he did. So, we can use it from [crates.io](https://crates.io/crates/eaglesong).

Reply:
published